reports/storing-large-real-time-data-streams-in-pod-using-LDES.md

```diff
@@ -0,0 +1,198 @@
+# Storing large real-time data streams in pod using LDES
+
+The corresponding challenge is [#82](https://github.com/SolidLabResearch/Challenges/issues/82),
+which contributes to scenario [#16](https://github.com/SolidLabResearch/Challenges/issues/16).
+
+## Problem
+<!--
+You can reuse the pitch of the challenge, but check if you need to make changes.
+For example, it might happen that the approved solution does more than what the original pitch requested.
+-->
+
+Data streams are becoming omnipresent and 
+are a crucial component in many use cases. 
+Storing streams in a low-cost file-based Web environment could be done using 
+[Linked Data Event Streams](https://semiceu.github.io/LinkedDataEventStreams/) (LDES). 
+However, pushing large volumes of high volatile data into a Solid-based LDES is not possible because 
+the [current solution](https://github.com/woutslabbinck/SolidEventSourcing) does the partitioning of the data 
+after all data has been retrieved, 
+instead of in a streaming fashion. 
+This crashes the [Community Solid Server](https://github.com/CommunitySolidServer/CommunitySolidServer) 
+due to the high load on the server 
+when repartitioning large amounts of data.
+We use data from the [DAHCC dataset](https://dahcc.idlab.ugent.be/) for this challenge,
+which contains data streams describing the behaviour of various patients.
+The streams contain over 100.000 events.
+
+We want a streaming Solid-based LDES connector that replays data and 
+partitions this data in a streaming fashion when retrieving the data, 
+instead of needing to wait till the whole dataset is received.
+Besides avoid the high load on the server,
+replaying allows to mimic the real-time behaviour of data streams,
+even though the data is historical data.
+This allows to showcase how solutions can process live data streams and
+how they can handle different data rates.
+
+TODO: @Stijn Does this build on [this](https://github.com/woutslabbinck/SolidEventSourcing)?
```

Comment:
@pheyvaer Correct. The reason it's not mentioned as a 'real' dependency is because of historical reasons, to be honest. Wout's (initial) implementation was not in a library-fashion. Therefore, we had to pick'n'pop (fork) the pieces of code that we needed specifically for our use-case. I'm not sure if the current state of his SolidEventSourcing allows it to be used as a library yet. In case it is, we might need some refactoring then on our side.